[2022-09-13 20:16:42,086] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify scheduled__2022-09-09T00:00:00+00:00 [queued]>
[2022-09-13 20:16:42,108] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify scheduled__2022-09-09T00:00:00+00:00 [queued]>
[2022-09-13 20:16:42,110] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:42,113] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-13 20:16:42,114] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:42,133] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): notify> on 2022-09-09 00:00:00+00:00
[2022-09-13 20:16:42,143] {standard_task_runner.py:52} INFO - Started process 225 to run task
[2022-09-13 20:16:42,149] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'notify', 'scheduled__2022-09-09T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/pre_parser.py', '--cfg-path', '/tmp/tmpdwzpfrjr', '--error-file', '/tmp/tmp0wffrt_q']
[2022-09-13 20:16:42,152] {standard_task_runner.py:80} INFO - Job 21: Subtask notify
[2022-09-13 20:16:42,250] {task_command.py:369} INFO - Running <TaskInstance: test_dag.notify scheduled__2022-09-09T00:00:00+00:00 [running]> on host 5d92941617fc
[2022-09-13 20:16:42,351] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=notify
AIRFLOW_CTX_EXECUTION_DATE=2022-09-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-09T00:00:00+00:00
[2022-09-13 20:16:42,367] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-13 20:16:42,370] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hello!"']
[2022-09-13 20:16:42,412] {subprocess.py:85} INFO - Output:
[2022-09-13 20:16:42,416] {subprocess.py:92} INFO - Hello!
[2022-09-13 20:16:42,419] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-13 20:16:42,469] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=notify, execution_date=20220909T000000, start_date=20220913T201642, end_date=20220913T201642
[2022-09-13 20:16:42,537] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-13 20:16:42,607] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
